cdap-client-tests/src/test/java/co/cask/cdap/client/MetadataHttpHandlerTestRun.java

```diff
@@ -1185,6 +1185,14 @@ public void testSearchResultSorting() throws Exception {
     } catch (BadRequestException e) {
       // expected
     }
+
+    // search with cursor for relevance sort
+    try {
+      searchMetadata(NamespaceId.DEFAULT, "search*", EnumSet.allOf(MetadataSearchTargetType.class), null, 1, "cursor");
```

Comment:
Use Assert.fail at the end of the try block, unless its ok for the call to now throw an exception